internal/handlers/telegram/handler.go

```diff
@@ -184,3 +188,26 @@ func documentHandler(tg *Client, u *tgbotapi.Message) {
 
 	tg.sendToIrc(formatted)
 }
+
+/*
+locationHandler receivers a location object from Telegram, and sends
```

Comment:
"receivers" -- should be "receives"

This is the only issue, so still approving 👍